nebula/src/nebula/faq/routes/faq_routes.py

```
@@ -0,0 +1,12 @@
+from fastapi import APIRouter, HTTPException
```

Ruff (0.12.2)

`fastapi.HTTPException` imported but unused

Remove unused import: `fastapi.HTTPException`

(F401)